content/docs/services/cloud-gov-identity-provider.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov identity provider
 name: "cloud-gov-identity-provider"
-description: "Manage client credentials for authenticating cloud.gov users in your app"
+description: "Authenticate cloud.gov users in your app"
```

Comment:
I believe Tock currently uses this service - to log into Tock, you go through the cloud.gov auth flow. Not sure what else / what the other capabilities are.